modules/cluster-logging-collector-external.adoc

```diff
@@ -134,16 +208,22 @@ For example:
   </store>
 ----
 
+. Create a ConfigMap named `secure-forward` in the `openshift-logging` namespace from the configuration file:
++
+----
+$ oc create configmap secure-forward --from-file=secure-forward.conf.yaml  -n openshift-logging
+----
+
 . Optionally, create a secret named `secure-forward` in the `openshift-logging` namespace.
 +
 ----
-$ oc create secret secure-forward --from-file=<arbitrary-name-of-key>=<file-name-of-ssl-private-key> --from-file=<arbitrary-name-of-cert>=<file-name-of-ssl-cert>
+$ oc create secret generic secure-forward --from-file=<arbitrary-name-of-key>=<file-name-of-ssl-private-key> --from-file=<arbitrary-name-of-cert>=<file-name-of-ssl-cert>
 ----
 +
 For example:
 +
 ----
-$ oc create secret secure-forward --from-file=tls.key=ca-for-es/logging-es.key --from-file=tls.crt=/etc/ocp-forward/ca-bundle.crt
+$ oc create secret generic secure-forward --from-file=tls.key=ca-for-es/logging-es.key --from-file=tls.crt=/etc/ocp-forward/ca-bundle.crt
```

Comment:
It is better to only import the necessory secrets.  How about use the following example:
oc create secret generic secure-forward --from-file=ca-bundle.crt=ca-for-fluent-receiver/ca.crt  --from-literal=shared_key=fluent-receiver

Reply:
https://github.com/openshift/openshift-docs/blob/3c1882126ceadff51d57bf48041c242a1c91fa82/modules/cluster-logging-collector-external.adoc
1. Please modify: 

- [ ] @type secure_forward   --->  <store>  @type forward

- [ ] oc create configmap secure-forward --from-file=secure-forward.conf.yaml  -n openshift-logging

   ---> 
    oc create configmap secure-forward --from-file=secure-forward.conf  -n openshift-logging

- [ ]  We need a final step to refresh fluentd  pod to apply the secure-forward secret and secury-forward configmap.
         oc delete pod --selector logging-infra=fluentd